classes/PodsUI.php

```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_start">
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
-									<?php echo PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field ); ?>
+									<?php
+										// Prevent p div issues.
+										echo str_replace(
+											array(
+												'<div',
+												'</div>',
+											),
+											array(
+												'<span',
+												'</span>',
+											),
+											PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field )
+										);
+									?>
 
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_end">
 										to
 									</label>
 								<?php
-									echo PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field );
+									// Prevent p div issues.
+									echo str_replace(
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field )
+									);
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 4c6dbb5f07d476df78ea4abab6dd6ccc</sub>